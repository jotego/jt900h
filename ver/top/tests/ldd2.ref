00000011 - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
0000babe - 00000000 - 00000000 - 00000000
-- index registers --
00000079 - 00000059 - 00000000 - 00000100
SR = f842
