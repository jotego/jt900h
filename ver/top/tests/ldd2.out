00000011 - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
0000babe - 00000000 - 00000000 - 00000000
-- index registers --
0000007d - 0000005d - 00000000 - 00000000
SR = f842
